framework/src/meshgenerators/RenameBlockGenerator.C

```diff
@@ -23,39 +23,11 @@ RenameBlockGenerator::validParams()
 
   params.addRequiredParam<MeshGeneratorName>("input", "The mesh we want to modify");
 
-  params.addDeprecatedParam<std::vector<SubdomainID>>(
-      "old_block_id",
-      "Elements with this block number will be given the new_block_number or "
-      "new_block_name.  You must supply either old_block_id or old_block_name.  "
-      "You may supply a vector of old_block_id, in which case the new_block "
-      "information must also be a vector.",
-      "Use 'old_block' instead of 'old_block_id'.");
-  params.addDeprecatedParam<std::vector<SubdomainName>>(
-      "old_block_name",
-      "Elements with this block name will be given the new_block_number or "
-      "new_block_name.  You must supply either old_block_id or old_block_name.  "
-      "You may supply a vector of old_block_name, in which case the new_block "
-      "information must also be a vector.",
-      "Use 'old_block' instead of 'old_block_name'.");
-  params.addDeprecatedParam<std::vector<SubdomainID>>(
-      "new_block_id",
-      "Elements with the old block number (or name) will be given this block "
-      "number.  If the old blocks are named, their names will be passed onto the "
-      "newly numbered blocks.",
-      "Use 'new_block' instead of 'new_block_id'.");
-  params.addDeprecatedParam<std::vector<SubdomainName>>(
-      "new_block_name",
-      "Elements with the old block number (or name) will be given this block "
-      "name.  No change of block ID is performed, unless multiple old blocks are "
-      "given the same name, in which case they are all given the first old block "
-      "number.",
-      "Use 'new_block' instead of 'new_block_name'.");
```

Comment:
this change will have to go in another PR, I m pretty sure Kylee s mesh could not be done without selecting the id independently

Reply:
yes she has to. The new ones wont let you set ids directly

Reply:
Okay, I'll put this back for this PR. I agree we need another PR. It's not good practice to rely on deprecatedParam. There should be better solution for it